src/hume/core/websocket.py

```diff
@@ -0,0 +1,10 @@
+from tkinter.messagebox import Message
+import typing
+
+
+OnOpenCloseHandlerType = typing.Union[typing.Callable[[], None], typing.Callable[[], typing.Awaitable[None]]]
+
+MessageT = typing.TypeVar('MessageT')
+OnMessageHandlerType = typing.Union[typing.Callable[[MessageT], None], typing.Callable[[MessageT], typing.Awaitable[None]]]
+
+OnErrorHandlerType = typing.Union[typing.Callable[[Exception], None], typing.Callable[[Exception], typing.Awaitable[None]]]
```

Comment:
```suggestion
import typing


OnOpenCloseHandlerType = typing.Union[typing.Callable[[], None], typing.Callable[[], typing.Awaitable[None]]]
MessageT = typing.TypeVar('MessageT')
OnMessageHandlerType = typing.Union[typing.Callable[[MessageT], None], typing.Callable[[MessageT], typing.Awaitable[None]]]
OnErrorHandlerType = typing.Union[typing.Callable[[Exception], None], typing.Callable[[Exception], typing.Awaitable[None]]]
```
Removes unused import (Message from tkinter)